articles/install-guide/index.md

```diff
@@ -17,237 +17,11 @@ Learn how to install the Microsoft Quantum Development Kit (QDK), so that you ca
 - a host application (written in Python or a .NET language) that runs quantum operations written in Q#
 - tools to facilitate your development
 
-Depending on your chosen development environment, there are different installation steps. Choose your environment from the sections below.
+Q# programs are often paired with a host program written in a .NET language (typically C#) or Python. This allows us to call quantum operations from inside the classical program.
```

Comment:
```suggestion
Q# programs are often paired with a host program written in a .NET language, typically C# or Python. This allows us to call quantum operations from inside a classical program.
```

Reply:
Without the parenthesis looks like Python is a .NET language when is not.